Vic2ToHoI4Tests/MapperTests/FlagsToIdeas/FlagsToIdeasMapperTests.cpp

```diff
@@ -0,0 +1,20 @@
+#include "Mappers/FlagsToIdeas/FlagsToIdeasMapper.h"
+#include "Mappers/FlagsToIdeas/FlagsToIdeasMapperFactory.h"
+#include "gtest/gtest.h"
+
+
+
+TEST(Mappers_FlagsToIdeas_FlagToIdeaMapperTests, unmatchedFlagReturnEmptyOptional)
+{
+	const auto theMapper = Mappers::FlagsToIdeasMapper::Factory().importFlagsToIdeaMapper();
+
+	ASSERT_FALSE(theMapper->getIdea("unmatched_flag"));
```

Comment:
Heh, remember that discussion about how the one true proper way was to do ASSERT_EQ against std::nullopt because that's what's being returned? :P I started doing that in tests and you revert to false without telling me? :)

Reply:
Heh.